pkg/release-controller/release_info.go

```diff
@@ -239,7 +239,12 @@ func (r *ExecReleaseInfo) ChangeLog(from, to string, json bool) (string, error)
 		}
 		return "", fmt.Errorf("could not generate a changelog: %v", msg)
 	}
-
+	if isJson {
+		var changeLog ChangeLog
+		if err := json.Unmarshal(out.Bytes(), &changeLog); err != nil {
+			return "", err
+		}
+	}
```

Comment:
I'm not convinced that this is happening very often in the production environment...
If we hit this logic, then it's probably worth a log entry.